railties/test/commands/dbconsole_test.rb

```diff
@@ -5,8 +5,15 @@
 require "rails/command"
 require "rails/commands/dbconsole/dbconsole_command"
 require "active_record/database_configurations"
+require "active_support/testing/method_call_assertions"
+
+require "active_record/connection_adapters/sqlite3_adapter"
```

Comment:
Instead of load all adapter for test that only test the sqlite3 adapter for example, we should move those tests to the adapter folders.

Reply:
The test is in Railties so that would mean moving the tests to Active Record, right?

Reply:
Moved the tests to Active Record testing `.dbconsole` instead of the command itself.